### PR TITLE
[Files] - fixes and improvements

### PR DESCRIPTION
**Pull request by GuyAfik**
## Description
* Error improvements for the `files` package
* Fixes an issue where if failing to load a text based file when reading file directly from memory and not from path, the SDK crashes.

### CODE REVIEW

File: demisto_sdk/commands/common/files/structured_file.py [hunk 1]
```diff
@@ -30,7 +30,13 @@ def as_default(cls, **kwargs):
         return instance
 
     def load(self, file_content: bytes) -> Any:
-        return self.handler.load(StringIO(super().load(file_content)))
+        file_content_as_text = super().load(file_content)
```
Comment (ilaner) [inline demisto_sdk/commands/common/files/structured_file.py:33]: what happens if this fails?

File: demisto_sdk/commands/common/files/ini_file.py [hunk 2]
```diff
@@ -16,7 +17,14 @@ def is_model_type_by_path(cls, path: Path) -> bool:
 
     def load(self, file_content: bytes) -> ConfigParser:
         config_parser = ConfigParser(allow_no_value=True)
-        config_parser.read_string(super().load(file_content))
+        file_content_as_text = super().load(file_content)
```
Comment (ilaner) [inline demisto_sdk/commands/common/files/ini_file.py:20]: what happens if this fails?

**Review by ilaner — APPROVED** [review]
nice job, see comments

File: demisto_sdk/commands/common/files/ini_file.py (hunk 2 above)
Comment (GuyAfik) [reply demisto_sdk/commands/common/files/ini_file.py:20]: will raise `FileLoadError` which is what we want, it means we couldn't load the object into the requested object.

**Pull request merged**